### PR TITLE
ko: CR2 Chromium translations

### PR DESCRIPTION
**Pull request by henrif75**
Chromium translations from CR v.2

#262 #1463

### CODE REVIEW

File: po/ko.po [hunk 1]
```diff
@@ -15026,34 +15028,46 @@ msgid ""
...
+"밋 위치 1223636 이후, 2023년 11월에 해당) 어떤 플랫폼이나 빌드 플래그 집합도 "
+"괜찮습니다."
 
 #: src/chromium/setup.md
 msgid ""
 "(A component, debug build is recommended for quickest iteration time. This "
 "is the default!)"
 msgstr ""
+"반복 시간을 최소화하려면 구성요소 디버그 빌드를 사용하는 것이 좋습니다. 이는 "
+"기본값입니다."
```
Comment (jiyongp) [inline po/ko.po:15051]:
```suggestion
"개발 속도를 빠르게 하고 싶다면 디버그 빌드를 사용하는 것이 좋습니다. 이게 "
"기본값입니다."
```

File: po/ko.po [hunk 2]
```diff
@@ -15026,34 +15028,46 @@ msgid ""
...
 
 #: src/chromium/setup.md
 msgid ""
 "See [How to build Chromium](https://www.chromium.org/developers/how-tos/get-"
 "the-code/) if you aren't already at that point. Be warned: setting up to "
 "build Chromium takes time."
 msgstr ""
+"아직 빌드하지 않았다면 [Chromium 빌드 방법](https://www.chromium.org/"
+"developers/how-tos/get-the-code/)을 참고하세요. 주의: Chromium 빌드 설정에는 "
+"시간이 걸립니다."
```
Comment (jiyongp) [inline po/ko.po:15061]:
```suggestion
"아직 빌드하지 않았다면 [Chromium 빌드 방법](https://www.chromium.org/"
"developers/how-tos/get-the-code/)을 참고하세요. 주의: Chromium 빌드 환경 셋업은 "
"시간이 걸립니다."
```

File: po/ko.po [hunk 3]
```diff
@@ -15062,44 +15076,63 @@ msgid ""
...
 #: src/chromium/cargo.md
+#, fuzzy
 msgid ""
 "The Rust community typically uses `cargo` and libraries from [crates.io]"
 "(https://crates.io/). Chromium is built using `gn` and `ninja` and a curated "
 "set of dependencies."
 msgstr ""
+"Rust 커뮤니티는 일반적으로 [crates.io](https://crates.io/)의 `cargo` 및 라이"
+"브러리를 사용합니다. Chromium은 `gn`, `ninja`, 선별된 종속 항목 집합을 사용하"
+"여 빌드됩니다."
```
Comment (jiyongp) [inline po/ko.po:15092]:
```suggestion
"Rust 커뮤니티는 일반적으로 [crates.io](https://crates.io/)의 `cargo` 및 라이"
"브러리를 사용합니다. Chromium은 세심하게 선별된 의존성들과 함께 `gn`, `ninja`을 이용하"
"여 빌드됩니다."
```

File: po/ko.po [hunk 4]
```diff
@@ -15062,44 +15076,63 @@ msgid ""
 "If you don't have time to complete a certain part, don't worry: you can "
 "catch up in the next slot."
 msgstr ""
+"과정의 이 부분에는 서로 연계되는 일련의 연습문제가 있습니다. 마지막에 한꺼번"
+"에 하지 않고 과정 전반에 연습문제가 흩어져 있습니다. 특정 부분을 완료할 시간"
+"이 없더라도 걱정하지 마세요. 다음번에 따라잡을 수 있습니다."
 
 #: src/chromium/cargo.md
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:15084]:
```suggestion
```

File: po/ko.po [hunk 5]
```diff
@@ -15062,44 +15076,63 @@ msgid ""
...
 msgid ""
 "From here on we'll be focusing on `gn` and `ninja`, because this is how Rust "
 "code can be built into the Chromium browser. At the same time, Cargo is an "
 "important part of the Rust ecosystem and you should keep it in your toolbox."
 msgstr ""
+"여기서부터 `gn`과 `ninja`에 중점을 둡니다. Chromium 브라우저에 Rust 코드를 빌"
+"드할 수 있는 방법이기 때문입니다.  동시에 Cargo는 Rust 생태계에서 중요한 부분"
+"이므로 도구 상자에 보관해야 합니다."
```
Comment (jiyongp) [inline po/ko.po:15135]:
```suggestion
"여기서부터는 `gn`과 `ninja`에 집중할 겁니다. 그 도구들을 써야만 Chromium 브라우저에 Rust 코드를"
"빌드해서 넣을 수 있기 때문입니다.  이와 동시에, Cargo는 Rust 생태계에서 중요한 부분"
"이므로 Cargo에도 익숙해야 합니다."
```

File: po/ko.po [hunk 6]
```diff
@@ -15062,44 +15076,63 @@ msgid ""
...
 #: src/chromium/cargo.md
 msgid ""
 "Use `cargo`, trusting a [toolchain](https://rustup.rs/) and/or [crates "
 "downloaded from the internet](https://crates.io/)"
 msgstr ""
+"`cargo`를 사용하여 [도구 모음](https://rustup.rs/) 또는 [인터넷에서 다운로드"
+"한 크레이트](https://crates.io/)를 신뢰합니다."
 
 #: src/chromium/cargo.md
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:15127]:
```suggestion
```

File: po/ko.po [hunk 7]
```diff
@@ -15108,53 +15141,73 @@ msgstr "연습문제"
...
 #: src/chromium/cargo.md
 msgid ""
 "Discuss which tools, libraries, and groups of people need to be trusted when "
 "using `gn` and `ninja`, offline `cargo`, etc."
 msgstr ""
+"`gn` 및 `ninja`, 오프라인 `cargo` 등을 사용할 때 신뢰해야 하는 도구, 라이브러"
+"리, 사용자 그룹을 논의합니다."
 
 #: src/chromium/cargo.md
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:15163]:
```suggestion
```

File: po/ko.po [hunk 8]
```diff
@@ -15108,53 +15141,73 @@ msgstr "연습문제"
...
 #: src/chromium/cargo.md
 msgid ""
 "Notes/hints related to the first part of the exercise (\"scenarios where "
 "Cargo may offer an advantage\"):"
 msgstr ""
+"연습문제의 첫 부분과 관련된 참고사항/힌트('Cargo가 이점을 제공할 수 있는 시나"
+"리오'):"
 
 #: src/chromium/cargo.md
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:15182]:
```suggestion
```

File: po/ko.po [hunk 9]
```diff
@@ -15188,6 +15249,9 @@ msgid ""
 "typesystem) and running faster (as a compiled, rather than interpreted "
 "language)."
 msgstr ""
+"놀랍게도 업계에서 Rust가 명령줄 도구 작성으로 점점 인기를 얻고 있습니다.  라"
+"이브러리의 폭과 인체공학은 Python과 유사하지만 풍부한 typesystem 덕분에 더 강"
+"력하고 더 빠르게 실행됩니다(인터프리트 언어가 아닌 컴파일된 언어로)."
```
Comment (jiyongp) [inline po/ko.po:15254]:
```suggestion
"놀랍게도 업계에서 Rust가 명령줄 도구 작성으로 점점 인기를 얻고 있습니다.  라"
"이브러리의 다양함과 사용편의성은 Python과 유사하지만 풍부한 타입 시스템 덕분에 더 안"
"전하고 더 빠르게 실행됩니다(인터프리트 언어가 아닌 컴파일된 언어로)."
```

File: po/ko.po [hunk 10]
```diff
@@ -15196,94 +15260,121 @@ msgid ""
...
 
 #: src/chromium/cargo.md
 msgid ""
 "Disclaimer: a unique reason for using `cargo` was unavailability of `gn` "
 "when building and bootstrapping Rust standard library when building Rust "
 "toolchain."
 msgstr ""
+"면책 조항: `cargo`를 사용하는 독특한 이유는 Rust 도구 모음을 빌드할 때 Rust "
+"표준 라이브러리를 빌드하고 부트스트랩하는 경우 `gn`을 사용할 수 없기 때문입니"
+"다."
```
Comment (jiyongp) [inline po/ko.po:15298]:
```suggestion
"주의: 여기서 `cargo`를 사용하는 유일한 이유는 Rust 도구 모음을 빌드할 때 Rust "
"표준 라이브러리를 빌드하고 부트스트랩하는 경우 `gn`을 사용할 수 없기 때문입니"
"다."
```

File: po/ko.po [hunk 11]
```diff
@@ -15352,43 +15449,49 @@ msgid ""
...
 #: src/chromium/policy.md
 msgid "If this policy changes over time, the course will evolve to keep up."
-msgstr ""
+msgstr "시간이 지나면서 이 정책이 변경되면 교육 과정도 이에 맞게 변경됩니다."
 
 #: src/chromium/build-rules.md
-#, fuzzy
 msgid "Build rules"
-msgstr "빌드 규칙"
+msgstr "빌드 및 실행"
```
Comment (jiyongp) [inline po/ko.po:15474]:
```suggestion
msgstr "빌드 규칙"
```

File: po/ko.po [hunk 12]
```diff
@@ -15352,43 +15449,49 @@ msgid ""
...
 
 #: src/chromium/build-rules.md
 msgid ""
 "Rust code is usually built using `cargo`. Chromium builds with `gn` and "
 "`ninja` for efficiency --- its static rules allow maximum parallelism. Rust "
 "is no exception."
 msgstr ""
+"Rust 코드는 일반적으로 `cargo`를 사용하여 빌드됩니다. Chromium은 효율성을 위"
+"해 `gn` 및 `ninja`로 빌드됩니다. 정적 규칙은 최대 동시 로드를 허용합니다. "
+"Rust도 예외는 아닙니다."
```
Comment (jiyongp) [inline po/ko.po:15484]:
```suggestion
"Rust 코드는 일반적으로 `cargo`를 사용하여 빌드됩니다. Chromium은 효율성을 위"
"해 `gn` 및 `ninja`로 빌드됩니다. `ninja`의 정적인 빌드 규칙은 최대한의 병렬성을 가능하게 해 줍니다."
"Rust를 빌드할 때에도 예외는 아닙니다."
```

File: po/ko.po [hunk 13]
```diff
@@ -15480,85 +15619,117 @@ msgid ""
...
+"```"
 
 #: src/chromium/build-rules/vscode.md
 msgid ""
 "Types are elided in Rust code, which makes a good IDE even more useful than "
 "for C++. Visual Studio code works well for Rust in Chromium. To use it,"
 msgstr ""
+"Rust 코드에서는 유형이 생략되므로 우수한 IDE가 C++보다 훨씬 더 유용해집니다. "
+"Visual Studio Code는 Chromium의 Rust에서 잘 작동합니다. 사용하려면 다음을 실"
+"행합니다."
```
Comment (jiyongp) [inline po/ko.po:15643]:
```suggestion
"Rust 코드에서는 타입이 생략되므로 우수한 IDE가 C++보다 훨씬 더 유용해집니다. "
"Visual Studio Code는 Chromium의 Rust에서 잘 작동합니다. 사용하려면 다음을 실"
"행합니다."
```

File: po/ko.po [hunk 14]
```diff
@@ -15480,85 +15619,117 @@ msgid ""
...
-msgstr ""
+msgstr "`ln -s out/Debug/rust-project.json rust-project.json`"
 
 #: src/chromium/build-rules/vscode.md
 msgid ""
 "A demo of some of the code annotation and exploration features of rust-"
 "analyzer might be beneficial if the audience are naturally skeptical of IDEs."
 msgstr ""
+"잠재고객이 자연스럽게 IDE에 관해 회의적인 경우 rust-analyzer의 코드 주석 및 "
+"탐색 기능 중 일부에 관한 데모가 유용할 수 있습니다."
```
Comment (jiyongp) [inline po/ko.po:15669]:
```suggestion
"누군가가 IDE에 대해 회의적인 경우 rust-analyzer의 코드 주석 및 "
"탐색 기능 중 일부를 시연해 보면 생각을 바꾸는데 유용할 수 있습니다."
```

File: po/ko.po [hunk 15]
```diff
@@ -15640,19 +15835,27 @@ msgid ""
 "call C ABI functions. Later in the course, we'll connect C++ directly to "
 "Rust."
 msgstr ""
+"이 예는 최소 공통분모 상호 운용성 언어인 C로 귀결되기 때문에 일반적이지 않습"
+"니다. C++와 Rust 모두 기본적으로 C ABI 함수를 선언하고 호출할 수 있습니다. "
+"이 과정의 후반부에서 C++를 Rust에 직접 연결합니다."
```
Comment (jiyongp) [inline po/ko.po:15840]:
```suggestion
"이 예는, 어디서나 상호 운용 가능한 언어인 C로 귀결되기 때문에 일반적이지 않습"
"니다. C++와 Rust 모두 기본적으로 C ABI 함수를 선언하고 호출할 수 있습니다. "
"이 과정의 후반부에서 C++를 Rust에 직접 연결합니다."
```

File: po/ko.po [hunk 16]
```diff
@@ -15831,8 +16035,9 @@ msgid "Testing exercise"
 msgstr "Rust on Exercism"
 
 #: src/exercises/chromium/testing.md
+#, fuzzy
 msgid "Time for another exercise!"
-msgstr ""
+msgstr "연습문제 정보"
```
Comment (jiyongp) [inline po/ko.po:16040]:
```suggestion
msgstr "새로운 연습문제를 풀어봅시다!"
```

File: po/ko.po [hunk 17]
```diff
@@ -15831,8 +16035,9 @@ msgid "Testing exercise"
 msgstr "Rust on Exercism"
 
 #: src/exercises/chromium/testing.md
+#, fuzzy
```
Comment (jiyongp) [inline po/ko.po:16038]:
```suggestion
```

File: po/ko.po [hunk 18]
```diff
@@ -16415,67 +16728,71 @@ msgstr "러스트 생태계"
...
 
 #: src/chromium/adding-third-party-crates.md
 msgid "Small"
-msgstr ""
+msgstr "작게"
 
 #: src/chromium/adding-third-party-crates.md
 msgid "Transitive dependencies"
-msgstr ""
+msgstr "전이 종속 항목"
```
Comment (jiyongp) [inline po/ko.po:16751]:
```suggestion
msgstr "모든 종속성들"
```

File: po/ko.po [hunk 19]
```diff
@@ -16637,68 +16997,76 @@ msgid ""
 "design of `gn` and `ninja` which aim for static, deterministic, build rules "
 "to maximize parallelism and repeatability of builds."
 msgstr ""
+"빌드가 실패하는 경우 이는 빌드 시간에 임의의 작업을 실행하는 프로그램인 "
+"`build.rs` 때문일 수 있습니다. 이는 빌드의 동시 로드와 반복성을 최대화하기 위"
+"해 정적이고 결정론적인 빌드 규칙을 목표로 하는 `gn` 및 `ninja`의 설계와 근본"
+"적으로 상충됩니다."
```
Comment (jiyongp) [inline po/ko.po]:
```suggestion
"빌드가 실패하는 경우, 이는 빌드 시간에 임의의 작업을 실행하는 프로그램인 "
"`build.rs` 때문일 수 있습니다. 이는 빌드의 병렬성과 재현성을 최대화하기 위"
"해 정적인 빌드 규칙을 목표로 하는 `gn` 및 `ninja`의 설계와 근본"
"적으로 상충됩니다."
```

**Comment by henrif75** [comment]
Thank you @jiyongp!

**Pull request merged**